### PR TITLE
docs: common args inheritence from Flow API

### LINKED ISSUE #5353: Document the common args inheritence from Flow API to Executors and Gateway
Currently, flow API accepts all args of all entities.
These args will be propagated to other entities (gateway, executor) with some occasional exclusion.
We need to document this behavior and document what arguments are excluded

### PR DESCRIPTION
**Pull request by AnneYang720**
**Goals:**

- resolves #5353 
- [ ] check and update documentation. See [guide](https://github.com/jina-ai/jina/blob/master/CONTRIBUTING.md#-contributing-documentation) and ask the team.

### CODE REVIEW

File: docs/concepts/flow/basics.md [hunk 1]
```diff
@@ -301,6 +301,13 @@ You can control the access mode, storage class name and capacity of the attached
 `JINA_K8S_ACCESS_MODES`, `JINA_K8S_STORAGE_CLASS_NAME` and `JINA_K8S_STORAGE_CAPACITY`. Only the first volume will be considered to be mounted.
 ```
 
+```{tip}
```
Comment (alaeddine-13) [inline docs/concepts/flow/basics.md:304]: important instead of tip

File: docs/concepts/flow/basics.md [hunk 2]
```diff
@@ -301,6 +301,13 @@ You can control the access mode, storage class name and capacity of the attached
...
+All arguments received by `Flow()` API will be propagated to other entities (gateway, executor) with the following exclusions:
```
Comment (alaeddine-13) [inline docs/concepts/flow/basics.md:305]:
```suggestion
All arguments received by `Flow()` API will be propagated to other entities (gateway, executor) with the following exceptions:
```

**Review by alaeddine-13 — COMMENTED** [review]
I think the changes should be in create section https://docs.jina.ai/concepts/flow/basics/#create

**Comment by alaeddine-13** [comment]
Also update the docstring of flow constructor (we want to see such information in the flow constructor API)
https://docs.jina.ai/api/jina.orchestrate.flow.base/#jina.orchestrate.flow.base.Flow

File: docs/concepts/flow/basics.md [hunk 3]
```diff
@@ -18,6 +18,13 @@ f = Flow()
 ```
 ````
 
+```{important}
+All arguments received by `Flow()` API will be propagated to other entities (gateway, executor) with the following exceptions:
+
+- `uses` and `uses_with` won't be passed to gateway
```
Comment (JoanFM) [inline docs/concepts/flow/basics.md:24]:
```suggestion
- `uses` and `uses_with` won't be passed to Gateway
```

File: docs/concepts/flow/basics.md [hunk 4]
```diff
@@ -18,6 +18,13 @@ f = Flow()
...
+- `port`, `port_monitoring`, `uses` and `uses_with` won't be passed to executor
```
Comment (JoanFM) [inline docs/concepts/flow/basics.md:25]:
```suggestion
- `port`, `port_monitoring`, `uses` and `uses_with` won't be passed to Executor
```

File: docs/concepts/flow/basics.md [hunk 5]
```diff
@@ -18,6 +18,13 @@ f = Flow()
 ```
 ````
 
+```{important}
+All arguments received by `Flow()` API will be propagated to other entities (gateway, executor) with the following exceptions:
```
Comment (JoanFM) [inline docs/concepts/flow/basics.md:22]:
```suggestion
All arguments received by `Flow()` API will be propagated to other entities (Gateway, Executor) with the following exceptions:
```

File: jina/orchestrate/flow/base.py [hunk 6]
```diff
@@ -366,6 +366,11 @@ def __init__(
                 f = Flow.load_config('flow.yml')  # load Flow from YAML config
                 with f:
                     f.bock()  # serve Flow
+            
+        All arguments received by `Flow()` API will be propagated to other entities (gateway, executor) with the following exceptions:
+        
+        - `uses` and `uses_with` won't be passed to gateway
```
Comment (JoanFM) [inline jina/orchestrate/flow/base.py:372]: same here

File: jina/orchestrate/flow/base.py [hunk 7]
```diff
@@ -366,6 +366,11 @@ def __init__(
...
+        - `port`, `port_monitoring`, `uses` and `uses_with` won't be passed to executor
```
Comment (JoanFM) [inline jina/orchestrate/flow/base.py:373]: same here

**Review by JoanFM — CHANGES_REQUESTED** [review]

**Comment by alexcg1** [comment]
@scott-martens if you have time today could you handle this?

**Comment by scott-martens** [comment]
Documentation change looks good to me.

File: docs/concepts/flow/basics.md [hunk 8]
```diff
@@ -18,6 +18,13 @@ f = Flow()
 ```
 ````
 
+```{important}
+All arguments received by `Flow()` API will be propagated to other entities (Gateway, Executor) with the following exceptions:
```
Comment (alaeddine-13) [inline docs/concepts/flow/basics.md:22]:
```suggestion
All arguments received by {class}`~jina.Flow()` API will be propagated to other entities (Gateway, Executor) with the following exceptions:
```

File: jina/orchestrate/flow/base.py [hunk 9]
```diff
@@ -366,6 +366,11 @@ def __init__(
                 f = Flow.load_config('flow.yml')  # load Flow from YAML config
                 with f:
                     f.bock()  # serve Flow
+            
+        All arguments received by `Flow()` API will be propagated to other entities (Gateway, Executor) with the following exceptions:
```
Comment (alaeddine-13) [inline jina/orchestrate/flow/base.py:370]:
```suggestion
        All arguments received by {class}`~jina.Flow()` API will be propagated to other entities (Gateway, Executor) with the following exceptions:
```

**Review by JoanFM — APPROVED** [review]

**Pull request merged**